cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1189,6 +1189,25 @@ if (uring)
   endif()
 endif()
 
+#---Check for DAOS----------------------------------------------------------------
+if (daos AND daos_mock)
+  message(FATAL_ERROR "Options `daos` and `daos_mock` are mutually exclusive; only one of them should be specified.")
+endif()
+if (daos OR daos_mock)
```

Comment:
I am a tad bit confused :).  I though 'daos_mock' was meant to replace DOAS, why does it need DOAS?

Reply:
Thanks for spotting.  Fix separately checks for 'libuuid' (required by both, libdaos and libdaos_mock), and 'libdaos' (only for DAOS).